src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
-public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+import java.util.Random;
+
+class ColorSupplier {
+    private Random random = new Random();
+
+    String getRandomColor() {
```

Comment:
As you will not be able to access methods from another app package
```suggestion
    public String getRandomColor() {
```

Reply:
ok, done